custom_components/nodered/websocket.py

```diff
@@ -8,7 +8,7 @@
 from homeassistant.components import device_automation
 from homeassistant.components.conversation import (
     HOME_ASSISTANT_AGENT,
-    _get_agent_manager,
+    get_agent_manager,
 )
```

Comment:
This will allow it to work on older version of Home Assistant

```
from homeassistant.components.conversation import HOME_ASSISTANT_AGENT

try:
    from homeassistant.components.conversation import get_agent_manager
except ImportError:
    # _get_agent_manager was renamed to get_agent_manager in 2024.4.0
    from homeassistant.components.conversation import (
        _get_agent_manager as get_agent_manager,
    )
```

Reply:
Good call, added.